python/cuml/__init__.py

```diff
@@ -50,6 +50,8 @@
 from cuml.metrics.cluster.adjustedrandindex import adjusted_rand_score
 from cuml.metrics.regression import r2_score
 
+from cuml.naive_bayes.naive_bayes import MultinomialNB
```

Comment:
I assume this is just some cleanup you spotted while working on the PR and not related to the docstring generator, right?

Reply:
yeah, I noticed it when I saw that NB was not in the docs, and wasn't here either